AutomatedTesting/Gem/PythonTests/AWS/common/resource_mappings.py

```diff
@@ -86,7 +87,7 @@ def clear_output_keys(self) -> None:
             resource_mappings = json.load(file_content)
 
         resource_mappings[AWS_RESOURCE_MAPPINGS_ACCOUNT_ID_KEY] = ''
-        resource_mappings[AWS_RESOURCE_MAPPINGS_REGION_KEY] = 'us-west-2'
+        resource_mappings[AWS_RESOURCE_MAPPINGS_REGION_KEY] = constants.AWS_REGION
```

Comment:
what is the value of constants.AWS_REGION?

Reply:
us-east-1 currently. We want to have the tests run under the default AWS region.